api_client/python/timesketch_api_client/sigma.py

```diff
@@ -95,3 +98,41 @@ def file_relpath(self):
             return ''
 
         return sigma_data.get('file_relpath', '')
+
+    def from_rule(self, rule_uuid):
+        """Get a Sigma object from a rule uuid.
+
+        Args:
+            rule_uuid: Id of the sigma rule.
+
+        """
+        # TODO: Write docstring
+        self.rule_uuid = rule_uuid
+        # TODO: not sure which one is the better one
+        self._resource_uri = f'sigma/{rule_uuid}'
+        self.resource_uri = f'sigma/{rule_uuid}'
+        super().__init__(
+            api=self.api, resource_uri=self._resource_uri)
+
+        self.lazyload_data()
+
+    def from_text(self, rule_text):
+        """Get a Sigma object from a rule text.
+
+        Args:
+            rule_text: Rule text to be parsed.
+
+        """
+        resource_url = '{0:s}/sigma_by_text/'.format(self.api.api_root)
+        data = {'title': 'Get_Sigma_by_text', 'content': rule_text}
+        response = self.api.session.post(resource_url, data=data)
+        print(response.__dict__)
```

Comment:
you do here ..

```
data = error.get_response_json(response, logger)
```

and that way you can get the response JSON back, which is a dict with the results.

Reply:
which I see you do in the next line... so skip this print

Reply:
ack